pkg/config/config_test.go

```diff
@@ -108,5 +130,24 @@ config_url: "https://gist.githubusercontent.com/mudler/ab26e8dd65c69c32ab2926857
 			Expect(c.Bundles[0].Targets[0]).To(Equal("package:utils/edgevpn"))
 			Expect(c.String()).ToNot(Equal(cc))
 		})
+
+		It("keeps header", func() {
+
+			var cc string = `
+config_url: "https://gist.githubusercontent.com/mudler/7e3d0426fce8bfaaeb2644f83a9bfe0c/raw/77ded58aab3ee2a8d4117db95e078f81fd08dfde/testgist.yaml"
```

Comment:
we need to setup an http server here to get rid of those, but yep.. not related to this PR here